community/kernel/pom.xml

```diff
@@ -134,7 +134,7 @@ the relevant Commercial Agreement.
         <configuration>
           <dotGitDirectory>${git.directory}</dotGitDirectory>
           <failOnNoGitDirectory>false</failOnNoGitDirectory>
-          <useNativeGit>true</useNativeGit>
+          <useNativeGit>false</useNativeGit>
```

Comment:
I think it caused problems for @jimwebber, for some reason, so @digitalstain said he'd make this change and see if it caused any problems. Apparently it hasn't for anyone in the core-edge team, since it's still there.